src/components/Navigator.vue

```diff
@@ -95,7 +95,10 @@ export default {
   computed: {
     // gets the paths for each parent in the breadcrumbs
     parentTopicReferences({ references, parentTopicIdentifiers }) {
-      return parentTopicIdentifiers.map(identifier => references[identifier]);
+      return parentTopicIdentifiers
+        .map(identifier => references[identifier])
```

Comment:
Not a blocker, but it may be slightly more optimized to use a single `reduce` loop as opposed to the two loops needed for a `map` and subsequent `filter`.